### PR TITLE
Improve Readme

### PR DESCRIPTION
**Pull request by alrik**
* project summary
* features
* roadmap

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,7 +1,133 @@
...
+
+### Message Persistence
+Message history can be enabled and fine tuned on channel level.
+Once enabled, all messages published to the channel will be kept for a configured amount of time.
+Persisted messages can be retrieved via API history resource.
+
+<details>
+<summary>Code sample</summary>
+
+```js
```
Comment (derlarsen) [inline README.md]: Using the SDK in code samples might be a bit misleading as the SDK is not part of this repo. Maybe explanations work better with HTTP/cURL requests?

File: README.md [hunk 2]
```diff
@@ -1,7 +1,133 @@
...
+
+### Custom Ids
+It’s just a little detail that manifests the seamless integration into your stack:
+All API resources can be operated with optional custom ids.
+Here is an example of how working with custom ids in our nodejs sdk looks like:
+
+<details>
+<summary>Code sample</summary>
+
+```js
```
Comment (derlarsen) [inline README.md]: See above

File: README.md [hunk 3]
```diff
@@ -1,7 +1,133 @@
...
+  <img src="media/realmq-platform-logo.svg" alt="RealMQ Logo" />
+  <br><br>
+</h1>
+
+![RealMQ Platform Build Status](https://drone.rmq.ovh/api/badges/RealMQ/realmq-platform/status.svg)
 [![FOSSA Status](https://app.fossa.io/api/projects/git%2Bgithub.com%2Frealmq%2Frealmq-platform.svg?type=shield)](https://app.fossa.io/projects/git%2Bgithub.com%2Frealmq%2Frealmq-platform?ref=badge_shield)
 
+# RealMQ :rocket: Open Source Real-time Messaging Platform
+
+[RealMQ](https://realmq.com) is a free and open source backbone for real-time telecommunication.
```
Comment (derlarsen) [inline README.md]: `telecommunication` => `communication`

File: README.md [hunk 4]
```diff
@@ -1,7 +1,133 @@
...
+
+// Or get an id auto generated
+const subscription = await realmq.subscriptions.create();
+subscription.id // => 20f62e87-e689-4a11-bcf3-a78026fffd85
+
+```
+
+</details>
+
+### Multi-Tenancy / Projects
```
Comment (baranga) [inline README.md]: IMHO the headline should be `Realms` and the text explaining that term with multi tenancy and projects.

**Pull request merged**